test/cppsuite/configs/background_compact_default.txt

```diff
@@ -0,0 +1,61 @@
+# Configuration for background_compact.
+duration_seconds=60,
+cache_size_mb=10000,
+
+timestamp_manager=
+(
+    enabled=true,
+    oldest_lag=5,
+    stable_lag=5,
+    op_rate=1s
+),
+
+workload_manager=
+(
+    populate_config=
+    (
+        collection_count=3,
+        key_count_per_collection=1000000,
+        thread_count=3,
+        key_size=10,
+        value_size=1000
+    ),
+
+    insert_config=
+    (
+        thread_count=3,
+        op_rate=10ms,
+        ops_per_transaction=(min=5,max=20),
+        key_size=10,
+        value_size=1000
+    ),
+
+    remove_config=
+    (
+        op_rate=2s,
+        ops_per_transaction=( max=1,min=0),
```

Comment:
Extra whitespace

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/commit/b21630dc0d16a3e94d7d729cc9d519ab60c84c7e.

Reply:
This commit does not contain changes in the configs

Reply:
My bad! [Fix configs.](https://github.com/wiredtiger/wiredtiger/pull/9691/commits/0014dfa45ae7c27ab3a958f59f8131232b05ea77)